src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -131,12 +142,14 @@
                             Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
 
     <AddSourceToNuGetConfig NuGetConfigFile="$(NuGetConfigFile)"
-                            SourceName="$(SourceBuiltShippingNuGetSourceName)"
-                            SourcePath="$(ArtifactsShippingPackagesDir)" />
+                            SourceName="%(RepositoryReferenceInfo.ShippingSourceName)"
```

Comment:
Why not use `SourceBuildSources` to drive adding sources here?

Reply:
`SourceBuildSources` is just feed names. The call to `AddSourceToNuGetConfig` needs to map the feed name to the path.